packages/carbon/src/internals/FieldsHandler.ts

```diff
@@ -185,4 +186,24 @@ export class FieldsHandler extends Base {
 		}
 		return result
 	}
+
+	public getFileUpload(
+		key: string,
+		required?: false
+	): ResolvedFile[] | undefined
+	public getFileUpload(key: string, required: true): ResolvedFile[]
+	public getFileUpload(key: string, required = false) {
```

Comment:
I'm thinking, the method for text inputs is `getText`, should this be `getFile` for consistently, or the other way around? (I don't mind `getFileUpload`, just something that came to mind)